data/config_files/full_dataset_random.json

```diff
@@ -0,0 +1,121 @@
+{
```

Comment:
rename it to example_config_random_occlusion
the full dataset term is used in conjunction with the paper